packages/devtools-modules/index.js

```diff
@@ -0,0 +1,15 @@
+const Services = require("./client/shared/shim/Services").Services;
+const SplitBox = require("./client/shared/components/splitter/SplitBox")
+// const SplitBoxCSS = require("./client/shared/components/splitter/SplitBox.css")
+const rep = require("./client/shared/components/reps/rep");
+// const repCSS = require("./client/shared/components/reps/reps.css");
+const Grip = require("./client/shared/components/reps/grip");
+
+module.exports = {
+  Services,
+  SplitBox,
+  // SplitBoxCSS,
+  rep,
+  // repCSS,
```

Comment:
I copied this into the debugger.html components dir because the requires didn't work... we can debug this now or later...